lib/resources/r4/related_person.yaml

```diff
@@ -61,9 +61,19 @@ fields:
     - display: Millennium Person Relationship Types
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-351-person-relationship-types
-    - display: Millennium Person Relationships
-      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
-      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
+
+- name: relationship relation extension
+  type: CodeableConcept
```

Comment:
The type should be Extension.

Reply:
removed because of  https://github.com/cerner/fhir.cerner.com/pull/624#issuecomment-826635553